src/runtime/pipeline/pipeline_function.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+#ifndef TVM_RUNTIME_PIPELINE_PIPELINE_FUNCTION_H_
+#define TVM_RUNTIME_PIPELINE_PIPELINE_FUNCTION_H_
+#include <tvm/runtime/module.h>
+#include <tvm/runtime/packed_func.h>
+#include <tvm/runtime/registry.h>
+
+#include <fstream>
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "pipeline_struct.h"
+namespace tvm {
+namespace runtime {
+/*!
+ * \brief The class that executes the pipeline logic,it is used to initialize the thread pool,
+    execute and schedule pipeline tasks, allocate and manage memory, etc.
+ */
+class PipelineFunction {
+ public:
+  /*!
+   * \brief Use the information of mod_config to create graph executor list.
+   * \param mod_config The configuration information generated by the library export library
+   *  function call.
+   */
+  std::vector<Module> PipelineCreateGraphExecutors(const ModuleConfig& mod_config);
+  /*!
+   * \brief Initialize the pipeline.
+   * \param modules The list of graph executor module.
+   * \param pipeline_config The dependency information of each graph executor module.
+   * \param mod_config The config information generated by the export library function call.
+   */
+  size_t PipelineInit(Array<Module> modules, const PipelineConfig& pipeline_config,
+                      const ModuleConfig& mod_config);
+
+ private:
+  /*!\brief The list of graph executors.*/
+  std::vector<Module> graph_executors_;
```

Comment:
Is calling `Module` `graph_executors_` correct?

Reply:
changed into graph_modules_